container-engine-lib/lib/database_accessors/enclave_db/file_artifacts_db/file_artifacts_db.go

```diff
@@ -0,0 +1,166 @@
+package file_artifacts_db
+
+import (
+	"encoding/json"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/database_accessors/consts"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/database_accessors/enclave_db"
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
+	bolt "go.etcd.io/bbolt"
+)
+
+type FileArtifactPersisted struct {
+	enclaveDb *enclave_db.EnclaveDB
+	data      *fileArtifactData
+}
+
+type fileArtifactData struct {
+	ArtifactNameToArtifactUuid map[string]string
+	ShortenedUuidToFullUuid    map[string][]string
+	ArtifactContentMd5         map[string][]byte
+}
+
+var (
+	takenIpAddressBucketName = []byte("file-artifact")
+	simpleKey                = []byte("my-key")
```

Comment:
nit: could we use another key name? something more semantic

Reply:
Done!